### PR TITLE
biz-dev job post

### PR DESCRIPTION
**Pull request by vykster**
Open to review by anyone, how does the formatting look? Tagging @angela-tran since I know she's been doing a lot of the website work lately.

### CODE REVIEW

**Comment by vykster** [comment]
Screenshot 2022-12-02 at 12 48 58 PM

oh shows up as not open. going to check the code for more info.

File: _jobs/executive-assistant.md
Comment (angela-tran) [inline _jobs/executive-assistant.md:4]: You can add `open: true` here to make it show up in the "Accepting applications" section

Comment (angela-tran) [reply _jobs/executive-assistant.md:4]:
```suggestion
open: true
---
```

Comment (vykster) [reply _jobs/executive-assistant.md:4]: woah this GH interface is cool. but ha, yes just found that in the code! thanks!

File: _jobs/executive-assistant.md
Comment (angela-tran) [inline _jobs/executive-assistant.md:22]: Should the first sentence in each of these list items be bolded like in the Google doc?

> **Write proposal materials.** You will be responsible for writing accurate and concise bids, often 6 to 20 pages, yourself. You will be writing about software development, data engineering, public policy...

**Comment by angela-tran** [comment]
Just one formatting thing I noticed. Otherwise, it looks great!

File: _jobs/executive-assistant.md
Comment (vykster) [reply _jobs/executive-assistant.md:22]: I noticed we didn't use bold in any of the other job descriptions on the site so skipping it for now.

Comment (angela-tran) [reply _jobs/executive-assistant.md:22]: Ok, sounds good 👍

**Review by angela-tran — APPROVED** [review]

**Comment by vykster** [comment]
ok, did I screw up and do that wrong? process is wait for the approval checkmark before merging?

**Comment by angela-tran** [comment]
That's generally the process, but no worries - that was fine!

**Pull request merged**
